### DISCUSSION TITLE
How to redirect URLs containing query parameters?

### DISCUSSION THREAD

**Discussion by Vacilando** [gatsbyjs/gatsby#34193] [Help]
How can we redirects URLs that contain query parameters?

We have URLs like https://www.example.com/index.php?x=1234 that we need to have redirected in Gatsby. (The "x" parameter is important there.)

I don't find any mention of query parameters in https://www.gatsbyjs.com/docs/reference/config-files/actions/#createRedirect 

Anybody an idea how to approach this?

**Comment by violy** [comment]
What is your hosting provider ?

Have you try to use full path (with `?query_params`) in `createRedirect` ?

**Comment by Vacilando** [comment]
Yes, we tried "/index.php?x=1234" in `createRedirect`. Result: 404. 

We use Gatsby Cloud. Before tried in Netlify, with their respective integration plugin. Did not work either.

I assume the query parameter has to be specified in another way to be recognized. But how?
